### ISSUE TITLE
DPO

### ISSUE THREAD

**Issue by yangliuIOC** [wdndev/tiny-llm-zh#15]
{'loss': 0.2374, 'grad_norm': 8.8805570602417, 'learning_rate': 2.281007963346787e-05, 'rewards/chosen': -0.2780037522315979, 'rewards/rejected': -7.671709060668945, 'rewards/accuracies': 0.8824999928474426, 'rewards/margins': 7.393705368041992, 'logps/rejected': -444.18621826171875, 'logps/chosen': -319.5079345703125, 'logits/rejected': -5.234597682952881, 'logits/chosen': -5.078985214233398, 'epoch': 0.01}
DPO阶段 rewards/chosen 全程是负数，还有reject 是负数
